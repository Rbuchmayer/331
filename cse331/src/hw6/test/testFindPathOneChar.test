LoadGraph oneCharOneBook oneCharOneBook.tsv
LoadGraph oneCharTwoBooks oneCharTwoBooks.tsv
FindPath oneCharOneBook A A
FindPath oneCharTwoBooks A A
FindPath oneCharOneBook A B
FindPath oneCharTwoBooks A B