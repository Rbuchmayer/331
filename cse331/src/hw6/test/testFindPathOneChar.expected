loaded graph oneCharOneBook
loaded graph oneCharTwoBooks
path from A to A:
path from A to A:
unknown character B
unknown character B